# IMUSetup block
# Block_Number 4224

std_msgs/Header header

# SBF block header including time header
BlockHeader block_header 

uint8   serial_port
float32 ant_lever_arm_x # m
float32 ant_lever_arm_y # m
float32 ant_lever_arm_z # m
float32 theta_x # deg
float32 theta_y # deg
float32 theta_z # deg